include/z64cutscene_commands.h

```diff
@@ -11,445 +11,266 @@
-    CMD_BBH(continueFlag, roll, frame), CMD_F(viewAngle), CMD_HH(xPos, yPos), CMD_HH(zPos, unused)
+#define CS_LIGHT_SETTING(setting, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7) \
+    CMD_HH(setting, startFrame), CMD_HH(endFrame, unused0), \
+    CMD_W(unused1), CMD_W(unused2), CMD_W(unused3), CMD_W(unused4), CMD_W(unused5), \
+    CMD_W(unused6), CMD_W(unused7), 0x00000000, 0x00000000, 0x00000000
 
 /**
- * ARGS
- *   s32 entries (e)
- * FORMAT
- *   00000009 eeeeeeee
- *   size = 0x8
+ * Declares a list of `CS_RUMBLE_CONTROLLER` entries.
  */
-#define CS_CMD_09_LIST(entries) CS_CMD_09, CMD_W(entries)
+#define CS_RUMBLE_CONTROLLER_LIST(entries) \
+    CS_CMD_RUMBLE_CONTROLLER, CMD_W(entries)
 
-/**
- * ARGS
- *   s16 unk (u), s16 startFrame (s), s16 endFrame (e), s16 unk2 (v), s16 unk3 (w), s16 unk4 (x)
- * FORMAT
- *   Capital U is Unused
- *   uuuussss eeeevvww xxUUUUUU
- *   size = 0xC
- */
-#define CS_CMD_09(unk, startFrame, endFrame, unk2, unk3, unk4, unused0, unused1) \
+#define CS_RUMBLE_CONTROLLER(unk, startFrame, endFrame, unk2, unk3, unk4, unused0, unused1) \
     CMD_HH(unk, startFrame), CMD_HBB(endFrame, unk2, unk3), CMD_BBH(unk4, unused0, unused1)
 
 /**
- * ARGS
- *   s32 cmdType (c), s32 entries (e)
- * FORMAT
- *   cccccccc eeeeeeee
- *   size = 0x8
- */
-#define CS_UNK_DATA_LIST(cmdType, entries) CMD_W(cmdType), CMD_W(entries)
-
-/**
- * ARGS
- *   s32 unk1 (a), s32 unk2 (b), s32 unk3 (c), s32 unk4 (d), s32 unk5 (e), s32 unk6 (f),
- *   s32 unk7 (g), s32 unk8 (h), s32 unk9 (i), s32 unk10 (j), s32 unk11 (k), s32 unk12 (l)
- * FORMAT
- *   aaaaaaaa bbbbbbbb cccccccc dddddddd eeeeeeee ffffffff gggggggg hhhhhhhh iiiiiiii jjjjjjjj kkkkkkkk llllllll
- *   size = 0x30
- */
-#define CS_UNK_DATA(unk1, unk2, unk3, unk4, unk5, unk6, unk7, unk8, unk9, unk10, unk11, unk12) \
-    CMD_W(unk1), CMD_W(unk2), CMD_W(unk3), CMD_W(unk4), CMD_W(unk5), CMD_W(unk6), \
-    CMD_W(unk7), CMD_W(unk8), CMD_W(unk9), CMD_W(unk10), CMD_W(unk11), CMD_W(unk12)
-
-/**
- * ARGS
- *   s32 cmdType (c), s32 entries (e)
- * FORMAT
- *   cccccccc eeeeeeee
- *   size = 0x8
+ * Declares a list of `CS_ACTOR_CUE` entries.
  */
-#define CS_NPC_ACTION_LIST(cmdType, entries) CMD_W(cmdType), CMD_W(entries)
+#define CS_ACTOR_CUE_LIST(cmdType, entries) \
+    CMD_W(cmdType), CMD_W(entries)
 
 /**
- * ARGS
- *   s16 npcAction (a), s16 startFrame (s), s16 endFrame (e),
- *   s16 rotX (u),      s16 rotY (v),       s16 rotZ (w),
- *   s32 startX (i),    s32 startY (j),     s32 startZ (k),
- *   s32 endX (l),      s32 endY (m),       s32 endZ (n),
- *   f32 normX (x),     f32 normY (y),      f32 normZ (z),
- * FORMAT
- *   aaaassss eeeeuuuu vvvvwwww iiiiiiii jjjjjjjj kkkkkkkk llllllll mmmmmmmm nnnnnnnn xxxxxxxx yyyyyyyy zzzzzzzz
- *   size = 0x30
+ * Defines a cue that an actor an actor can listen for. The actor can choose to use the position and rotatation data supplied to it.
```

Comment:
```suggestion
 * Defines a cue that an actor can listen for. The actor can choose to use the position and rotation data supplied to it.
```

Also did you mean
```suggestion
 * Defines a cue that an actor can listen for. The actor can choose to use the position and rotation data supplied to it for other purposes or not at all.
```

Reply:
I mean in general, that it is the actors choice whether it uses the data at all.
Just setting the position vector in the cue does not mean the actor will follow that position automatically. It has to actively read that data, and do the interpolation/setting of the position entirely on its own for it to do anything.

Reply:
So in other words, the pos and rotation are just a suggestion and not law the actor must abide by.
If theres a better way to phrase that, then we can change it

Reply:
Well if you are a new reader you may understand

> The actor can choose to use the position and rotation data supplied to it.

as simply "the position and rotation may be unused", not "it may not be position and rotation data"

That's why I suggested
> The actor can choose to use the position and rotation data supplied to it for other purposes or not at all.